ckanext/datastore/db.py

```diff
@@ -1310,8 +1313,7 @@ def make_private(context, data_dict):
 
 
 def make_public(context, data_dict):
-    log.info('Making resource {0} public'.format(
-        data_dict['resource_id']))
+    log.info('Making resource {resource_id!r} public'.format(**data_dict))
```

Comment:
Nope, just being consistent and quoting all of them. It's `UnicodeText` in the model...